src/miner_lora.erl

```diff
@@ -256,8 +255,36 @@ handle_json_data(#{<<"stat">> := Status} = Map, Gateway0, #state{gateways=Gatewa
     Gateway1 = Gateway0#gateway{status=Status},
     lager:info("got status ~p", [Status]),
     lager:info("Gateway ~p", [lager:pr(Gateway1, ?MODULE)]),
-    Mac = Gateway1#gateway.mac,
-    handle_json_data(maps:remove(<<"stat">>, Map), Gateway1, State#state{gateways=maps:put(Mac, Gateway1, Gateways)});
+    TimeInfo = case maps:find(<<"tacc">>, Status) of
+                   {ok, _} ->
+                       %% ok we have GPS information
+                       {gps, #gps_info_pb{
+                                horizontal_accuracy=maps:get(<<"eha">>, Status),
```

Comment:
This needs to take account of the claimed location, the resolution of that claim, as well as the *actual* gps location and the the gps horizontal accuracy.